package.json

```diff
@@ -36,9 +36,12 @@
   "devDependencies": {
     "babel": "^5.2.16",
     "babel-core": "^5.2.6",
+    "babel-eslint": "^4.1.3",
     "babel-loader": "^5.0.0",
     "babel-plugin-object-assign": "^1.0.1",
     "brfs": "^1.2.0",
+    "eslint": "^1.7.3",
```

Comment:
:question: Do we need to install these locally if there are no gulp tasks that make use of the dependencies?